### DISCUSSION TITLE
Please can we get a new release of Testcontainers?

### DISCUSSION THREAD

**Discussion by chriskilding** [testcontainers/testcontainers-java#6785] [General]
It's been a while since the last release of Testcontainers on Maven Central, and some useful features have accumulated in `main` since then. For example, I need the fix that makes Testcontainers work with Colima. Therefore a new release of Testcontainers would be greatly appreciated.

**Comment by eddumelendez** [comment]
Hi @chriskilding , there will be a release in a couple of weeks still. Which is the fix related to colima?

**Reply by chriskilding** [reply]
Hi, the fix in Testcontainers was simply to bump its version of the `docker-java` dependency to `3.3.0` (the actual bug fix was in docker-java).

This was done in commit 2c4caadc3d88bea2e75f9b83cbfe122995ca9bb5

**Reply by eddumelendez** [reply]
1.18.0 was released one month ago.
